experiments/react-native-uifabric/src/components/Persona/Persona.settings.ts

```diff
@@ -0,0 +1,12 @@
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { IPersonaType, personaName } from './Persona.types';
+
+export const settings: IComposeSettings<IPersonaType> = [
+  {
+    tokens: {
+      horizontalGap: 16,

```

Comment:
Should these values scale with the size of the persona?

Reply:
yes, it should.

Reply:
Added style function to calculate the gap based on persona size.